graql/reasoner/schema-queries.feature

```diff
@@ -460,6 +448,8 @@ Feature: Schema Query Resolution (Variable Types)
 
 
   # TODO: re-enable all steps when type generation is supported in resolution test framework (#75)
+#  TODO: disabled because down and side casting no longer allowed
```

Comment:
Same comment as above - if we're settled on banning type generation, we should delete these scenarios entirely

Reply:
deleted

Reply:
is this actually deleted?

Reply:
I mean though it was. Maybe rebase put it back (still trying that out instead of merge). Definitely deleted now.